hebrew/shame-on-you/boring/you-failed.md

```diff
@@ -0,0 +1,2 @@
+מצטער, אך נראה כאילו נכשלת בהרפתקאה שלך...
+[נסה שוב!](https://github.com/udacity/create-your-own-adventure/blob/master/hebrew/marshmallow.md)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.

Reply:
I think you've got a little thing to fix. Link ((https://github.com/udacity/create-your-own-adventure/blob/master/hebrew/marshmallow.md) is absolute link for web, you should change it to relative link for your current file. Maybe it should be hebrew/marshmallow.md or other link has no https://github.com/udacity. Fix and call bot cassey again. Good luck!!

Reply:
Alright, I fixed that too now.
Hope that's all of it :D